Gems/AWSGameLift/Code/AWSGameLiftClient/Source/Request/IAWSGameLiftInternalRequests.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/RTTI/RTTI.h>
+
+namespace Aws
+{
+    namespace GameLift
+    {
+        class GameLiftClient;
+    }
+} // namespace Aws
+
+namespace AWSGameLift
+{
+    //! IAWSGameLiftRequests
+    //! GameLift Gem internal interface which is used to fetch gem global GameLift client
```

Comment:
Internal indicates that this is something that developers shouldn't use. Is that the intent? Not sure what this request is really doing. Is it more IAWSGameLiftClientManagementRequests?

Reply:
it is internal request which is defined inside Source, so it is only visible inside gamelift.client gem